packages/gestalt/src/icons/index.js

```diff
@@ -206,6 +207,7 @@ export default {
   'fill-transparent': fillTransparent,
   filter,
   flag,
+  flash,
```

Comment:
Down below we have `'lightning-bolt-circle': lightningBoltCircle,` shouldn't this be called `lightning-bolt` to match the similarly looking icon?

Reply:
i can do that — i didn't want to confuse folks who looked in the icon generator (where it's called flash).  but i agree this makes more sense.

Reply:
Talked offline a bit — we use flash internally so it makes sense to keep it as flash here